test/users/adapters/driver/rest_api/router/get_users_endpoint_integration_test.py

```diff
@@ -0,0 +1,53 @@
+import typing
+
+import pytest
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from app.users.domain.factories import get_users_case
+from app.users.domain.ports.driver import GetUsersUseCase
+from app.users.domain import User
+from faker import Faker
+
+
+@pytest.fixture
+def client_factory(application: FastAPI) -> typing.Callable[..., TestClient]:
+    def factory(get_users_use_case: typing.Callable[..., GetUsersUseCase]):
+        application.dependency_overrides[get_users_case] = get_users_use_case
+        return TestClient(application)
+    yield factory
+    application.dependency_overrides = {}
+
+
+def generate_fake_user(faker: Faker) -> dict:
+    return {
+        "identifier": str(faker.uuid4()),
+        "name": faker.name(),
+        "lastname": faker.last_name(),
+        "age": faker.pyint(min_value=1, max_value=90),
+        "email": faker.email(),
+    }
```

Comment:
What do you think to create a Fixture that creates a User Factory in order to be used accros the tests of the User

Reply:
Thank you for this suggestion, I totally agree with this. I'm gonna create this one.